account_payment_order/security/payment_security.xml

```diff
@@ -5,7 +5,7 @@
 
 <record id="group_account_payment" model="res.groups">
     <field name="name">Accounting / Payments</field>
-    <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>
```

Comment:
Why do we need both?

Reply:
@sbidoul `base.user_root` access does not grant access to Admin since they are two different users in Odoo 12. I observed that it has been not as it pretty much everywhere in Odoo 12. https://github.com/odoo/odoo/commit/2f7c03d9ca5809b65dc6eaa778064668d34fcd2e